trax/supervised/configs/bert_glue_classification.gin

```diff
@@ -12,41 +12,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import trax.layers
-import trax.models
 import trax.data
-import trax.optimizers
-import trax.supervised.lr_schedules
-import trax.supervised.trainer_lib
 import trax.models.research.bert
-import trax.layers.metrics
 
 # See https://www.tensorflow.org/datasets/catalog/glue -- valid dataset_name
 # values are: glue/cola, glue/sst2, glue/mrpc, glue/qqp, glue/stsb, glue/mnli,
 # glue/qnli, glue/rte, glue/wnli. However, training on WNLI with this setup is
 # not recommended and will likely result in lower than baseline accuracy.
 
 
-dataset_name = 'glue/qnli'
+dataset_name = 'glue/mnli'
+
+include 'bert.gin'
 
 # Parameters for TFDS data pipeline:
 # ==============================================================================
 make_inputs.train_stream = [
   @train/data.TFDS(),
-  @data.Tokenize(),
-  @data.CreateBertInputs(),
-  @data.Shuffle(),
-  @data.PadToLength(),
-  @data.TruncateToLength(),
+  %bert_stream_core,
```

Comment:
duplicating `bert_stream_core` inline (i.e. as it was before) was fine ... because now mostly one will have to look back and forth between files to immediately see what the pipeline is ... but its OK for now.

Even commenting here that "bert_stream_core - tokenizes, creates berk inputs, shuffles, pads and truncates the sequence" will be good here.

Reply:
I see. I removed the bert_stream_core altogether and moved it inline as before. I thought that commenting with what it does might lead to inconsistencies if bert_stream_core is changed and comment is not